src/modules/fixedDimensions.js

```diff
@@ -27,3 +27,8 @@ export const FIXED_DIMENSIONS = {
         icon: OrgUnitIcon,
     },
 }
+
+export const filterFixedDimensions = dimensionIds =>
```

Comment:
I would rename this function so it indicates that it filters **out** the fixed dimensions.